static/app/gettingStartedDocs/javascript/react.tsx

```diff
@@ -69,6 +71,27 @@ export const steps = ({
       },
     ],
   },
+  {
+    language: 'html',
```

Comment:
when this is `javascript` the format looks a bit strange and there are no colors... we'll have to see support for npx

Reply:
how about language `bash`?

Reply:
`language` is for syntax highlighting of the `npx` command, right? Can we set something like `bash` or `sh` here? Maybe this is more accurate than html